plasma_framework/test/helpers/ife.js

```diff
@@ -162,6 +159,14 @@ function createInclusionProof(encodedTx, txUtxoPos) {
     };
 }
 
+function getOutputId(txBytes, utxoPos) {
```

Comment:
I would move this to `utils.js` to be with same file as other `computeXXXOutputId`